compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/model/types/BFiniteType.java

```diff
@@ -115,4 +116,16 @@ public void addValue(BLangExpression value) {
             nullable = true;
         }
     }
+
+    public Set<BType> getUniqueValues() {
```

Comment:
The name of the function is not accurate right? 

Also, this is like a specific scenario so I think this shouldn't be a method in BFiniteType.java. We can maybe introduce a method in the Types.java class itself instead.